apis/zalando.org/v1/types.go

```diff
@@ -67,7 +66,6 @@ const (
 
 // BackendAlgorithmType is the type of algorithm used for load balancing
 // traffic to a backend. This is only valid for backend type lb|service.
-// +kubebuilder:validation:Enum=roundRobin;random;consistentHash;powerOfRandomNChoices
```

Comment:
Moved kubebuilder enum annotation closer to the fields to have it along with description.